app/src/main/java/com/google/android/stardroid/activities/EditSettingsActivity.java

```diff
@@ -84,15 +84,32 @@ public void onCreate(Bundle savedInstanceState) {
   public void onStart() {
     super.onStart();
     analytics.trackPageView(Analytics.EDIT_SETTINGS_ACTIVITY);
-    Preference locationPreference = preferenceFragment.findPreference(LOCATION);
+    final Preference locationPreference = preferenceFragment.findPreference(LOCATION);
+    Preference latitudePreference = preferenceFragment.findPreference(LATITUDE);
+    final Preference longitudePreference = preferenceFragment.findPreference(LONGITUDE);
```

Comment:
Could you remove 'final' off this line for consistency with line 88? I see that you need it on line 87.